public/images/illustrations/illustration-404.svg

```diff
@@ -0,0 +1 @@
3-.8 5 0 3.5.6 7 1.9 10.2.6 1.6 1.5 3.1 2.5 4.3 1.1 1.2 2.5 1.9 3.8 1.5 1.2-.4 2.1-1.8 2.6-3.4.5-1.6.8-3.3.9-5 .2-3.5-.1-7-.7-10.4-.3-1.7-.8-3.4-1.4-5-.3-.8-.6-1.6-1.1-2.3-.4-.7-.9-1.3-1.4-1.6-.1 0-.2-.1-.4-.1h-.3s-.3.2-.4.5c-.3.6-.5 1.5-.6 2.3-.3 1.7-.3 3.5-.3 5.2 0 3.5.2 7.1.6 10.6 0 .3-.2.5-.5.5-.4.3-.6.1-.6-.2z"/><path class="st12" d="M268 121.2c.9-5.7-4.1-12.5-9.4-11.7-7 1.1-8.5 6.2-7.2 10.8.3 1-.5 2.4 0 3.3.5.8 1.7.1 2.1.9 1.9 3 5.5 4.4 8.9 3 1.6-.7 3.6-2.1 4.3-3.2"/><path class="st2" d="M267.5 121.1c.3-2.1-.2-4.3-1.2-6.1-1-1.9-2.5-3.5-4.4-4.4-.9-.4-1.9-.6-2.9-.5s-2.1.4-3 .9c-1.9.8-3.4 2.4-3.9 4.3-.3 1-.3 2-.2 3l.3 1.5c.1.2.2.7.2 1.1s-.1.7-.1 1c-.1.6-.2 1.1-.1 1.3h.5s.9 0 1.4.5l.3.3.1.2.2.3c.3.4.6.8 1 1.1 1.5 1.3 3.5 1.8 5.4 1.4.9-.2 1.8-.6 2.8-1.1.5-.2.9-.5 1.3-.8s.9-.6 1.2-.9c.2-.2.5-.2.7 0 .1.2.2.4.1.6-.6 1.2-1.5 2-2.4 2.7s-2 1.3-3.3 1.6c-2.5.6-5.2-.2-7.1-1.9-.5-.4-.9-.9-1.3-1.4l-.3-.4-.1-.2c.1.1.1 0-.2 0-.2 0-.6 0-1.2-.2-.5-.2-.9-.7-1-1.2-.2-.9 0-1.5.1-2.1 0-.3.1-.5.1-.7s0-.2-.1-.6c-.2-.6-.3-1.2-.3-1.8-.1-1.2 0-2.4.3-3.6s1-2.3 1.8-3.2c.9-.9 1.9-1.5 3-2 1.1-.4 2.2-.7 3.4-.8 1.2-.1 2.5.1 3.6.6 2.2 1 3.8 2.9 4.9 5s1.6 4.5 1.2 6.8c0 .3-.3.5-.6.4-.1-.2-.3-.4-.2-.7zm-14.9-8.4c1.7 2.7 4.4 4.4 7.3 5.7 2.9 1.3 6.1 2.1 9.3 3.1.3.1.4.4.3.6s-.3.4-.5.4c-1.7-.1-3.3-.4-5-.7-1.7-.4-3.3-.8-4.9-1.5s-3.1-1.5-4.5-2.7c-1.3-1.2-2.4-2.7-2.9-4.4-.1-.3.1-.5.3-.6.2-.2.4-.1.6.1z"/><path class="st13" d="M256.1 110.3c1.3-3-3.9-5.6-6.1-1.6-1.1 2-.1 4.9 2 5.1"/><path class="st2" d="M255.7 110.1c.2-1.1-.4-1.9-1.4-2.3-.9-.4-2-.2-2.7.4-.3.3-.7.8-.9 1.2-.2.4-.3.9-.2 1.5.1 1 .6 2 1.6 2.4.3.1.4.4.3.6-.1.2-.3.3-.5.3-.8 0-1.6-.4-2.2-1-.6-.6-.9-1.4-1.1-2.2-.1-.8-.1-1.6.2-2.4s.8-1.4 1.4-2c.6-.5 1.5-.9 2.3-1 .9-.1 1.7.1 2.4.5s1.3 1 1.7 1.8c.3.8.2 1.8-.2 2.4-.1.2-.5.3-.7.2 0 0-.1-.2 0-.4z"/><path class="st14" d="M254.7 120.1l.1.8"/><path class="st13" d="M254.3 125.4c.4.1.8 0 1.2-.1l.6-.3c.2-.1.3-.3.4-.4.1-.2.3-.2.5-.1.1.1.2.2.2.3 0 .4-.2.7-.4 1-.2.3-.5.4-.8.6-.3.1-.7.2-1 .1-.3 0-.7-.1-1-.4-.1-.1-.1-.3 0-.5 0-.2.1-.2.3-.2zm10.9-13.1c.4 3.3 7.8 4.7 7.8-1.4s-8.7-6.2-7.8 1.4z"/><path class="st2" d="M265.7 112.3c.1.9.9 1.6 1.8 1.9s1.9.3 2.7 0c.8-.3 1.3-1 1.6-1.8.2-.8.3-1.9.1-2.7-.2-.9-.7-1.6-1.4-2.1s-1.7-.6-2.5-.3-1.6 1-1.9 1.9c-.5 1-.5 2-.4 3.1zm-1 .1c-.1-1.1-.1-2.4.3-3.5s1.2-2.3 2.4-2.8 2.7-.5 3.9.1c1.2.6 2.1 1.8 2.5 3.1.2.6.2 1.3.2 1.9s-.1 1.3-.3 1.9c-.2.6-.6 1.3-1.1 1.7-.5.5-1.2.8-1.8 1-1.3.3-2.6.1-3.7-.5-.6-.3-1.1-.6-1.5-1.1-.5-.5-.9-1.2-.9-1.8z"/><path class="st15" d="M289.8 85.7l-4.9 12.6"/><path class="st2" d="M290.3 85.9c-.2 1.1-.5 2.2-.8 3.3-.4 1.1-.7 2.1-1.2 3.2l-1.3 3.1c-.5 1-1 2-1.6 3-.1.2-.5.3-.7.2-.2-.1-.3-.3-.2-.5.2-1.1.6-2.2.9-3.3.3-1.1.7-2.1 1.1-3.2s.8-2.1 1.3-3.1 1-2 1.6-3c.2-.2.5-.3.7-.1.2 0 .2.2.2.4z"/><path class="st13" d="M254.5 111.2c-3.2 0-1.8 1.5-1.8 1.5s-.9.3 0 1.5 6.6 5.5 10.8 6.6c4.3 1 3.8 1.5 4 1 .3-.5 1.3-3.8-1.2-7.4s-4.7-5.4-8-4.8l-3.8 1.6z"/><path class="st7" d="M310 205.6c-5.4.2-21.7 1.2-24.2 2.1-2.4.9-2.6 1.2.3 1.9 3 .7 23.8 0 23.8 0v-4h.1z"/></svg>
```

Comment:
It contains "drake-404" as title, we should remove titles, as it's already embedded in the page, or at least give a more generic one. :)